apps/website/src/components/notifications/NotificationSettingsForm.tsx

```diff
@@ -64,12 +64,11 @@ export function NotificationSettingsForm({
   return (
     <form
       onSubmit={submitHandler}
-      className={
-        "pb-2 transition-opacity " +
-        (enableSettings
+      className={`pb-2 transition-opacity ${
+        enableSettings
           ? ""
-          : "pointer-none cursor-default select-none opacity-50")
-      }
+          : "pointer-none cursor-default select-none opacity-50"
+      }`}
```

Comment:
Can we use `classes` here instead of a ternary?

Reply:
I don't understand completely what you mean. Could you give me an example?

Reply:
`classes` refers to the utility function at `@/utils/classes`, that can be used to concat classes from the arguments, filtering falsy values. so this could be

```tsx
className={classes(
    "pb-2 transition-opacity",
    enableSettings && "pointer-none cursor-default select-none opacity-50",
)}
```